tests/task/upgrade_recordings_imported_test.php

```diff
@@ -33,14 +33,20 @@
  */
 class upgrade_recordings_imported_test extends advanced_testcase {
     use testcase_helper_trait;
-
+    /**
+     * Setup for test
+     */
+    public function setUp(): void {
+        parent::setUp();
+        $this->require_mock_server();
```

Comment:
Maybe we should have `require_mock_server()` perform the `reset_mock()` call too.